src/Components/Indicators/PageIndicator/PageIndicator.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import "./PageIndicator.scss";
+
+const PageIndicator = ({ max, activeState }) => {
+  let dots = [];
+  //render indicator dots
+  for (let i = 0; i < max; ++i) {
+    dots.push(
+      <div
+        key={i}
+        className={`indicator-dot ${i === activeState ? "active" : ""}`}
+      ></div>
+    );
+  }
```

Comment:
We could do better with
```suggestion
  const dots = [...Array(max)].map((_, i) => (
    <div
      key={i}
      className={`indicator-dot ${i === activeState ? "active" : ""}`}
    />
  ));
```